ttnn/cpp/ttnn/operations/bernoulli/bernoulli_pybind.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "bernoulli_pybind.hpp"
+
+#include <optional>
+
+#include "bernoulli.hpp"
+#include "pybind11/decorators.hpp"
+
+namespace ttnn::operations::bernoulli {
+void bind_bernoulli_operation(py::module &module) {
+    auto doc =
+        R"doc(Bernoulli(input: Tensor, out: Optional[Tensor] = None, out_dtype: Optional[DataType] = None, memory_config: Optional[MemoryConfig] = None, compute_kernel_config: Optional[ComputeKernelConfig] = None) -> Tensor
+    Generates a tensor to draw binary random numbers (0 or 1) from a Bernoulli distribution.
+    This operation allows configuration of memory allocation using `memory_config` and computation settings via `compute_kernel_config`.
+
+    Args:
+        * :attr:`input`: The input tensor of probability values for the Bernoulli distribution.
+        * :attr:`out`: The output tensor.
+        * :attr:`out_dtype`: The output tensor dtype (default float32).
```

Comment:
Should be output.
out_dtype should be dtype.

Reply:
I've updated the param.